app/services/converter.py

```
@@ -10,12 +10,34 @@
+        raise RuntimeError("google-cloud-storage package is required for cloud mode")
+    try:
+        with open(settings.gcp_keypath, "r") as f:
+            credentials_info = json.load(f)
+        gcs_client = storage.Client.from_service_account_info(credentials_info)
+        logger.info(f"GCS client initialized for project: {gcs_client.project}")
+    except FileNotFoundError as exc:
+        logger.error(f"GCP key file not found: {settings.gcp_keypath}")
+        raise FileNotFoundError(f"GCP key file not found: {settings.gcp_keypath}") from exc
+    except Exception as e:
+        logger.error(f"Failed to initialize GCS client: {str(e)}")
+        raise RuntimeError(f"Failed to initialize GCS client: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)